client/app/states/dashboard/dashboard.e2e.js

```diff
@@ -0,0 +1,20 @@
+/* eslint angular/log: 0 */
+/* eslint no-console: 0 */
+/* eslint no-undef: 0 */
```

Comment:
hmmm i wonder if adding an `.eslintignore` to root with `**/*.e2e.js` would be better than specifying all the rules to ignore 🤔